include/kllvm/codegen/CreateTerm.h

```diff
@@ -19,71 +19,95 @@ class CreateTerm {
-      isAnywhereOwise(isAnywhereOwise) {}
-
-/* adds code to the specified basic block in the specified module which constructs
-   an llvm value corresponding to the specified KORE RHS pattern and substitution in the
-   specified definition, and returns the value itself, along with a boolean indicating whether the resulting term could be an injection. */
+      llvm::StringMap<llvm::Value *> &Substitution, KOREDefinition *Definition,
+      llvm::BasicBlock *EntryBlock, llvm::Module *Module, bool isAnywhereOwise)
+      : Substitution(Substitution)
+      , Definition(Definition)
+      , CurrentBlock(EntryBlock)
+      , Module(Module)
+      , Ctx(Module->getContext())
+      , isAnywhereOwise(isAnywhereOwise) { }
+
+  /* adds code to the specified basic block in the specified module which
+     constructs an llvm value corresponding to the specified KORE RHS pattern
+     and substitution in the
+     specified definition, and returns the value itself, along with a boolean
+     indicating whether the resulting term could be an injection. */
   std::pair<llvm::Value *, bool> operator()(KOREPattern *pattern);
   llvm::Value *createToken(ValueType sort, std::string contents);
-  /* creates a call instructin calling a particular llvm function, abstracting certain abi and calling convention details. 
-   * name: the nmae of the function to call in llvm
-   * returnCat: the value category of the return type of the function
-   * args: the arguments to pass to the functgion
-   * sret: if true, this is a function that returns a struct constant via the C abi, ie, 
-   * the function actually returns void and the return value is via a pointe. Note that this 
-   * can be set to true even if the function does not return a struct, in which case its value
-   * is ignored.
-   * load: if the function returns a struct via sret, then if load is true, we load the value 
-   * fastcc: true if we should use the fastcc calling convention
-   * returned from the function before returning it. */
-  llvm::Value *createFunctionCall(std::string name, ValueType returnCat, const std::vector<llvm::Value *> &args, bool sret, bool fastcc);
+  /* creates a call instructin calling a particular llvm function, abstracting
+   * certain abi and calling convention details. name: the nmae of the function
+   * to call in llvm returnCat: the value category of the return type of the
+   * function args: the arguments to pass to the functgion sret: if true, this
+   * is a function that returns a struct constant via the C abi, ie, the
+   * function actually returns void and the return value is via a pointe. Note
+   * that this can be set to true even if the function does not return a struct,
+   * in which case its value is ignored. load: if the function returns a struct
+   * via sret, then if load is true, we load the value fastcc: true if we should
+   * use the fastcc calling convention returned from the function before
+   * returning it. */
```

Comment:
It likes to reflow comments; I can either disable this behaviour (so comments will get ignored completely), or introduce line breaks between your list items so they get identified as distinct items.